### ISSUE TITLE
PHP Warning:  count(): Parameter must be an array or an object that implements Countable in mimePart.php on line 314

### ISSUE THREAD

**Issue by Aikhjarto** [roundcube/roundcubemail#7410]
I Just upgraded to roundcubemail 1.4.4 and the log is full of lines with the following error:
```
May 31 20:43:16 roundcube: PHP Warning:  count(): Parameter must be an array or an object that implements Countable in /srv/www/htdocs/roundcubemail/vendor/pear-pear.php.net/Mail_Mime/Mail/mimePart.php on line 314
```
Might be a regression of #6098
The testsystem is an openSUSE Leap 15.1 with PHP 7.2.5.

**Comment by johndoh** [comment]
Mail_Mime is a third party library used by Roundcube. This issue appears to been fixed in version 1.10.2 of that library. Please confirm your dependencies are up to date (run `php composer.phar install --no-dev`).

**Comment by Aikhjarto** [comment]
@johndoh I tried to do the update as you suggested but it failed.

First try: nothing happend, but an error and an update suggestion of composer.phar.
```
# php composer.phar install --no-dev
Warning: This development build of composer is over 30 days old. It is recommended to update it by running "composer.phar self-update" to get the latest version.
PHP Fatal error:  Cannot use 'Object' as class name as it is reserved in phar:///srv/www/htdocs/roundcubemail/composer.phar/vendor/justinrainbow/json-schema/src/JsonSchema/Constraints/Object.php on line 18

Fatal error: Cannot use 'Object' as class name as it is reserved in phar:///srv/www/htdocs/roundcubemail/composer.phar/vendor/justinrainbow/json-schema/src/JsonSchema/Constraints/Object.php on line 18
```
So I ran the update
```
# php composer.phar self-update
Updating to version 0a0291e913c8d71d37bb825c0400128a0f5a3e20.
    Downloading: 100%
Use composer self-update --rollback to return to version 68e910d92a87dde9c905c9d397fe2e5983ea32ab
```
But after update, I get a different error
```
# php composer.phar install --no-dev


  [RuntimeException]
  The PEAR repository has been removed from Composer 2.0
```

**Comment by alecpl** [comment]
Use mailing list or forum for help. The warning was fixed in Mail_mime 1.10.2.
